src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -318,6 +318,9 @@ async def _fetch_stream_responses(self) -> ResponseType:
             yield message
             message = await self._read()
 
+        # If the read operation failed, Core should explain why.
+        await self._raise_for_status()
```

Comment:
not really, when an error happens automatically the loop stops since a `cygrpc.EOF` is automatically propagated, is the caller who needs to double-check if the `cygrpc.EOF` happened because of an error of because there were no more messages to consume.